### ISSUE TITLE
Change Request: suppress PMR Pending tracked item

### ISSUE THREAD

**Issue by amylai-va** [department-of-veterans-affairs/va-mobile-app#9671]
### 1.) Requestor's Name

Amy Lai

### 2.) Requestor's Email

amy.lai2@va.gov

### 3.) VA Team

Benefits

### 4.) What is the name of the project or change?

Suppress PMR Pending requests

### 5.) Please explain what problem you are trying to solve.

PMR Pending is a tracked item that is added by the contract vendor that attempts to obtain private medial records (PMRs) from private physicians. Therefore, it should categorized as a request that VA is making to a 3rd party. However, these requests currently show as requests that a Veteran needs to take action on.

- This is the most common tracked item we show (>17k in the last 30 days)
- The current UI doesn't explain what is being asked of the Veteran
- If the private physician fails to provide the records, then regional office (RO) is responsible for notifying the Veteran. The RO sends a letter to the Veteran (no tracked item is created), and at that point, it is the responsibility of the Veteran to send the records to VBA within 30 days.

### 6.) Please explain how this change would work.

Assuming that PMR Pending requests also appear in the mobile app as something the Veteran needs to take an action item:
- **First:** Suppress the PMR Pending request. This will ensure the Veteran is not seeing inaccurate requests. `this request`
- **Next:** The VA.gov team will ask BGS to recategorize PMR requests, so they show up as 3rd party requests. Then, we will clarify what these requests are and how a Veteran might be able to help (even though it's a request for their private physician).

### 7.) Features Impacted

Claims, evidence requests

### 8.) Please include all relevant documentation links for this change request.

N/A

### 9.) Does your team have all the necessary materials (documents, designs, processes, analytics, APIs, etc) in place for Mobile to make these changes right now?

To be discussed

### 10.) Has this change been discussed with VAHB POs (Ryan or Don)?

No

### 11.) Has this change been agreed upon by all impacted VA Stakeholders?

Yes, Andrew Gray

### 12.) What is your ideal due date for this change request?

ASAP, since these requests are inaccurate

**Comment by DonMcCaugheyUSDS** [comment]
@amylai-va @jstrothman @CorySohrakoffUSDS  I'm not familiar with PMR Pending requests.  Where might one expect to find them in the mobile app?  This would help us check if we do in fact show them.

Also, what system & API endpoint do they come from?

**Comment by jstrothman** [comment]
@DonMcCaugheyUSDS A few notes about PMR Pending requests:

1. PMR Pending requests are a type of tracked item, which we believe appear as Claim File requests, e.g. [this screenshot (mural)](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1719581824942/975bd9a22e21177415cdb238525b3e1aac95a288?wid=0-1719586977159)
2. We get these requests via the Benefits Claims API
3. PMR Pending requests are incorrectly categorized by BGS as "needed_from_you" when they should be recategorized as "needed_from_others" 
4. We've submitted a request to BGS for re-categorization, however that can take a few months, and VBA has agreed that they should be suppressed until they can be recategorized as they are the most common tracked item we show (14k in last 23 days on VA.gov) and are conveying misinformation. In web CST, as an item "needed_from_you" they indicate something is needed from the Veteran by [x] date, and it's unclear what is needed from the Veteran
    * Instead of suppressing the PMR rquests, the web team decided to force them to appear along with tracked items categorized as "needed_from_others" in https://github.com/department-of-veterans-affairs/va.gov-team/issues/93456

I don't know if the mobile app displays evidence requests (tracked items) made to others, but these should not appear as requests for the Veteran.

**Comment by DonMcCaugheyUSDS** [comment]
@jstrothman Is it possible to make this change in vets-api where the code can be centralized in one place and shared by both mobile and web clients?

**Comment by jstrothman** [comment]
@DonMcCaugheyUSDS Yes, we made this request but because it can take months for the request to be addressed, and because we're currently incorrectly telling thousands of Veterans that _they_ need to take an action by a certain date, we'd like to suppress them in the UI until the change can be made in the endpoint. (See #4 in my comment from 3 weeks ago) 

The action request Veterans see is not only not something we're asking them to do, it's also non-specific ("private medical record request" with no details), so they are making calls to find out more about what is needed.

**Comment by DonMcCaugheyUSDS** [comment]
@jstrothman Yes, I know we're waiting for BGS to recategorize.  I'm specifically asking that we add the work-around code to vets-api where it can be shared by the vets-api endpoints that serve vets-website and the mobile client, rather than two separate fixes, one in vets-website and one in the mobile app.  There is technical risk with implementing this fix twice -- veterans may not see consistent data between web and mobile.  We prefer to keep this kind of business logic centralized in one place in vets-api whenever possible (and it is possible in this case).

Turn-around time to get a change like this into the mobile app is at least a month: one sprint for engineering work and one sprint for testing and release to the app stores.  Then it can take around a week for mobile users to receive the update automatically (and there are around 10% of users who don't update their app regularly).

Getting a change into vets-api is much faster: a new version is deployed every business day, so we're only limited by how quickly an engineer can complete this change.  If the change is backwards compatible (this would be) then all mobile and web users receive it immediately after vets-api is deployed.

**Comment by amylai-va** [comment]
There was some discussion [in this slack thread](https://dsva.slack.com/archives/C018V2JCWRJ/p1727200042004819?thread_ts=1727143329.932729&cid=C018V2JCWRJ) about whether it would be possible for the stopgap measure to be applied in only one place. Adding it to the ticket for reference and continued discussion:
- @CorySohrakoffUSDS: is there any reason why we wouldn't want to do this in the BE (like BenefitsClaimsController)?
- Jacob: We haven't explored a BE solution for the interim fix. It's a small amount of work to recategorize this in our app on the FE as a stop-gap interim solution, I'm unsure if it's worth exploring an application agnostic BE interim solution if, in the long run, all applications will receive the tracked item request with the correct values from LH. (defer to engineers)
- Jerek: We can do it in BenefitsClaimsController but that controller is only used by [VA.gov](http://va.gov/) so not sure that there is an advantage there

@DonMcCaugheyUSDS would you or someone familiar with the mobile implementation be able to chime in on that?

**Comment by CorySohrakoffUSDS** [comment]
Amy, the BMT team has already implemented a frontend solution for web. The question for us is do we want them to refactor that into a BE solution that mobile can use. I think the answer is yes, considering the 4+ week lead time the mobile app needs for release.

It's also not a good practice for us to have the same logic in multiple places if we can help it.

**Comment by BeccaTupaj** [comment]
This change request has been added to the Core Mobile App Team backlog and going forward will be tracked using the epic #10484 . Closing this Change Request

**Comment by CorySohrakoffUSDS** [comment]
@BeccaTupaj The backend work has already been completed. I believe it got sign off from the mobile team [here](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/10334), so I think we can consider this one completed.
